2009-05-06 21:17:37,656 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-06 21:17:38,343 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-06 21:17:38,343 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-06 21:21:36,328 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-06 21:21:36,984 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-06 21:21:36,984 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 00:15:39,984 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 00:15:41,234 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 00:15:41,234 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 00:20:18,937 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 00:20:19,593 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 00:20:19,593 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 00:21:36,359 [SwingWorker-pool-2-thread-7] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 00:21:36,390 [SwingWorker-pool-2-thread-7] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 00:21:36,390 [SwingWorker-pool-2-thread-7] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 00:21:38,234 [SwingWorker-pool-2-thread-9] INFO  org.mesh4j.sync.translator.MessageTranslator - Resource Bundle for key <rdfXml> does not exist.
2009-05-07 00:21:38,250 [SwingWorker-pool-2-thread-9] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - java.lang.IllegalArgumentException: rdfXml
org.mesh4j.sync.validations.MeshException: java.lang.IllegalArgumentException: rdfXml
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.convertXMLToRow(HibernateContentAdapter.java:126)
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.save(HibernateContentAdapter.java:65)
	at org.mesh4j.sync.adapters.split.SplitAdapter.update(SplitAdapter.java:108)
	at org.mesh4j.sync.SyncEngine.importItems(SyncEngine.java:191)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:122)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:50)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:54)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:47)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: rdfXml
	at org.mesh4j.sync.validations.Guard.throwsArgumentException(Guard.java:36)
	at org.mesh4j.sync.payload.schema.rdf.RDFInstance.buildFromRDFXml(RDFInstance.java:79)
	at org.mesh4j.sync.payload.schema.rdf.RDFSchema.createNewInstanceFromRDFXML(RDFSchema.java:253)
	at org.mesh4j.sync.adapters.hibernate.mapping.HibernateToRDFMapping.convertXMLToRow(HibernateToRDFMapping.java:59)
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.convertXMLToRow(HibernateContentAdapter.java:124)
	... 16 more
2009-05-07 00:21:42,265 [SwingWorker-pool-2-thread-10] INFO  org.mesh4j.sync.translator.MessageTranslator - Resource Bundle for key <rdfXml> does not exist.
2009-05-07 00:21:42,281 [SwingWorker-pool-2-thread-10] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - java.lang.IllegalArgumentException: rdfXml
org.mesh4j.sync.validations.MeshException: java.lang.IllegalArgumentException: rdfXml
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.convertXMLToRow(HibernateContentAdapter.java:126)
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.save(HibernateContentAdapter.java:65)
	at org.mesh4j.sync.adapters.split.SplitAdapter.update(SplitAdapter.java:108)
	at org.mesh4j.sync.SyncEngine.importItems(SyncEngine.java:191)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:122)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:50)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:54)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:47)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: rdfXml
	at org.mesh4j.sync.validations.Guard.throwsArgumentException(Guard.java:36)
	at org.mesh4j.sync.payload.schema.rdf.RDFInstance.buildFromRDFXml(RDFInstance.java:79)
	at org.mesh4j.sync.payload.schema.rdf.RDFSchema.createNewInstanceFromRDFXML(RDFSchema.java:253)
	at org.mesh4j.sync.adapters.hibernate.mapping.HibernateToRDFMapping.convertXMLToRow(HibernateToRDFMapping.java:59)
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.convertXMLToRow(HibernateContentAdapter.java:124)
	... 16 more
2009-05-07 00:44:17,703 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 00:44:18,265 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 00:44:18,265 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 00:44:20,062 [SwingWorker-pool-2-thread-3] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - rdfSchema
java.lang.IllegalArgumentException: rdfSchema
	at org.mesh4j.sync.validations.Guard.argumentNotNull(Guard.java:15)
	at org.mesh4j.sync.adapters.msexcel.MsExcelRDFSyncAdapterFactory.createSyncAdapter(MsExcelRDFSyncAdapterFactory.java:39)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMsExcelAdapter(SyncAdapterBuilder.java:166)
	at org.mesh4j.ektoo.controller.MsExcelUIController.createAdapter(MsExcelUIController.java:93)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:348)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:40)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2009-05-07 00:44:21,828 [SwingWorker-pool-2-thread-4] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - rdfSchema
java.lang.IllegalArgumentException: rdfSchema
	at org.mesh4j.sync.validations.Guard.argumentNotNull(Guard.java:15)
	at org.mesh4j.sync.adapters.msexcel.MsExcelRDFSyncAdapterFactory.createSyncAdapter(MsExcelRDFSyncAdapterFactory.java:39)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMsExcelAdapter(SyncAdapterBuilder.java:166)
	at org.mesh4j.ektoo.controller.MsExcelUIController.createAdapter(MsExcelUIController.java:93)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:348)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:40)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2009-05-07 00:46:24,953 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 00:46:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 00:46:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 00:49:13,234 [SwingWorker-pool-2-thread-6] INFO  org.mesh4j.sync.translator.MessageTranslator - Resource Bundle for key <rdfXml> does not exist.
2009-05-07 00:49:13,250 [SwingWorker-pool-2-thread-6] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - java.lang.IllegalArgumentException: rdfXml
org.mesh4j.sync.validations.MeshException: java.lang.IllegalArgumentException: rdfXml
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.convertXMLToRow(HibernateContentAdapter.java:126)
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.save(HibernateContentAdapter.java:65)
	at org.mesh4j.sync.adapters.split.SplitAdapter.add(SplitAdapter.java:54)
	at org.mesh4j.sync.SyncEngine.importItems(SyncEngine.java:188)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:122)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:50)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:79)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:72)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: rdfXml
	at org.mesh4j.sync.validations.Guard.throwsArgumentException(Guard.java:36)
	at org.mesh4j.sync.payload.schema.rdf.RDFInstance.buildFromRDFXml(RDFInstance.java:79)
	at org.mesh4j.sync.payload.schema.rdf.RDFSchema.createNewInstanceFromRDFXML(RDFSchema.java:253)
	at org.mesh4j.sync.adapters.hibernate.mapping.HibernateToRDFMapping.convertXMLToRow(HibernateToRDFMapping.java:59)
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.convertXMLToRow(HibernateContentAdapter.java:124)
	... 16 more
2009-05-07 00:50:32,703 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 00:50:33,265 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 00:50:33,265 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 00:50:34,453 [SwingWorker-pool-2-thread-3] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - rdfSchema
java.lang.IllegalArgumentException: rdfSchema
	at org.mesh4j.sync.validations.Guard.argumentNotNull(Guard.java:15)
	at org.mesh4j.sync.adapters.msexcel.MsExcelRDFSyncAdapterFactory.createSyncAdapter(MsExcelRDFSyncAdapterFactory.java:39)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMsExcelAdapter(SyncAdapterBuilder.java:166)
	at org.mesh4j.ektoo.controller.MsExcelUIController.createAdapter(MsExcelUIController.java:93)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:348)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:40)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2009-05-07 00:51:53,953 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 00:51:54,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 00:51:54,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 02:03:55,015 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 02:03:55,484 [SwingWorker-pool-2-thread-1] ERROR org.mesh4j.sync.utils.SqlDBUtils - Access denied for user 'root'@'localhost' (using password: YES)
java.sql.SQLException: Access denied for user 'root'@'localhost' (using password: YES)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1055)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.mesh4j.sync.utils.SqlDBUtils.getTableNames(SqlDBUtils.java:32)
	at org.mesh4j.ektoo.ui.MySQLUI.setList(MySQLUI.java:421)
	at org.mesh4j.ektoo.ui.MySQLUI$11$1.doInBackground(MySQLUI.java:358)
	at org.mesh4j.ektoo.ui.MySQLUI$11$1.doInBackground(MySQLUI.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2009-05-07 02:04:02,609 [SwingWorker-pool-2-thread-2] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 02:04:02,734 [SwingWorker-pool-2-thread-2] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 02:04:02,734 [SwingWorker-pool-2-thread-2] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 02:04:04,171 [SwingWorker-pool-2-thread-4] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - rdfSchema
java.lang.IllegalArgumentException: rdfSchema
	at org.mesh4j.sync.validations.Guard.argumentNotNull(Guard.java:15)
	at org.mesh4j.sync.adapters.msexcel.MsExcelRDFSyncAdapterFactory.createSyncAdapter(MsExcelRDFSyncAdapterFactory.java:39)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMsExcelAdapter(SyncAdapterBuilder.java:166)
	at org.mesh4j.ektoo.controller.MsExcelUIController.createAdapter(MsExcelUIController.java:96)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:348)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:42)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2009-05-07 02:05:35,828 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 02:05:36,390 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 02:05:36,390 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 02:27:46,734 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 02:27:47,281 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo
2009-05-07 02:27:47,296 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: ektoo_sync_info
2009-05-07 02:38:01,531 [SwingWorker-pool-2-thread-5] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - Getting database metadata
org.hibernate.exception.SQLGrammarException: Getting database metadata
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:67)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getMetaData(AbstractMetaDataDialect.java:64)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.caseForSearch(AbstractMetaDataDialect.java:163)
	at org.hibernate.cfg.reveng.dialect.JDBCMetaDataDialect.getTables(JDBCMetaDataDialect.java:22)
	at org.hibernate.cfg.reveng.JDBCReader.processTables(JDBCReader.java:476)
	at org.hibernate.cfg.reveng.JDBCReader.readDatabaseSchema(JDBCReader.java:78)
	at org.hibernate.cfg.reveng.JDBCReader.readDatabaseSchema(JDBCReader.java:860)
	at org.hibernate.cfg.JDBCBinder.readDatabaseSchema(JDBCBinder.java:115)
	at org.hibernate.cfg.JDBCBinder.readFromDatabase(JDBCBinder.java:88)
	at org.hibernate.cfg.JDBCMetaDataConfiguration.readFromJDBC(JDBCMetaDataConfiguration.java:42)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.autodiscoveryMappings(HibernateSyncAdapterFactory.java:197)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createMappings(HibernateSyncAdapterFactory.java:141)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createHibernateAdapter(HibernateSyncAdapterFactory.java:58)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMySQLAdapter(SyncAdapterBuilder.java:140)
	at org.mesh4j.ektoo.controller.MySQLUIController.createAdapter(MySQLUIController.java:104)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:336)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:57)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'ektoo'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.Util.getInstance(Util.java:381)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.hibernate.connection.DriverManagerConnectionProvider.getConnection(DriverManagerConnectionProvider.java:110)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getConnection(AbstractMetaDataDialect.java:122)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getMetaData(AbstractMetaDataDialect.java:61)
	... 24 more
2009-05-07 02:38:10,406 [SwingWorker-pool-2-thread-6] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - Getting database metadata
org.hibernate.exception.SQLGrammarException: Getting database metadata
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:67)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getMetaData(AbstractMetaDataDialect.java:64)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.caseForSearch(AbstractMetaDataDialect.java:163)
	at org.hibernate.cfg.reveng.dialect.JDBCMetaDataDialect.getTables(JDBCMetaDataDialect.java:22)
	at org.hibernate.cfg.reveng.JDBCReader.processTables(JDBCReader.java:476)
	at org.hibernate.cfg.reveng.JDBCReader.readDatabaseSchema(JDBCReader.java:78)
	at org.hibernate.cfg.reveng.JDBCReader.readDatabaseSchema(JDBCReader.java:860)
	at org.hibernate.cfg.JDBCBinder.readDatabaseSchema(JDBCBinder.java:115)
	at org.hibernate.cfg.JDBCBinder.readFromDatabase(JDBCBinder.java:88)
	at org.hibernate.cfg.JDBCMetaDataConfiguration.readFromJDBC(JDBCMetaDataConfiguration.java:42)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.autodiscoveryMappings(HibernateSyncAdapterFactory.java:197)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createMappings(HibernateSyncAdapterFactory.java:141)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createHibernateAdapter(HibernateSyncAdapterFactory.java:58)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMySQLAdapter(SyncAdapterBuilder.java:140)
	at org.mesh4j.ektoo.controller.MySQLUIController.createAdapter(MySQLUIController.java:104)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:336)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:57)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'ektoo'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.Util.getInstance(Util.java:381)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.hibernate.connection.DriverManagerConnectionProvider.getConnection(DriverManagerConnectionProvider.java:110)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getConnection(AbstractMetaDataDialect.java:122)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getMetaData(AbstractMetaDataDialect.java:61)
	... 24 more
2009-05-07 02:38:11,859 [SwingWorker-pool-2-thread-7] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - Getting database metadata
org.hibernate.exception.SQLGrammarException: Getting database metadata
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:67)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getMetaData(AbstractMetaDataDialect.java:64)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.caseForSearch(AbstractMetaDataDialect.java:163)
	at org.hibernate.cfg.reveng.dialect.JDBCMetaDataDialect.getTables(JDBCMetaDataDialect.java:22)
	at org.hibernate.cfg.reveng.JDBCReader.processTables(JDBCReader.java:476)
	at org.hibernate.cfg.reveng.JDBCReader.readDatabaseSchema(JDBCReader.java:78)
	at org.hibernate.cfg.reveng.JDBCReader.readDatabaseSchema(JDBCReader.java:860)
	at org.hibernate.cfg.JDBCBinder.readDatabaseSchema(JDBCBinder.java:115)
	at org.hibernate.cfg.JDBCBinder.readFromDatabase(JDBCBinder.java:88)
	at org.hibernate.cfg.JDBCMetaDataConfiguration.readFromJDBC(JDBCMetaDataConfiguration.java:42)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.autodiscoveryMappings(HibernateSyncAdapterFactory.java:197)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createMappings(HibernateSyncAdapterFactory.java:141)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createHibernateAdapter(HibernateSyncAdapterFactory.java:58)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMySQLAdapter(SyncAdapterBuilder.java:140)
	at org.mesh4j.ektoo.controller.MySQLUIController.createAdapter(MySQLUIController.java:104)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:336)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:57)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'ektoo'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.Util.getInstance(Util.java:381)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.hibernate.connection.DriverManagerConnectionProvider.getConnection(DriverManagerConnectionProvider.java:110)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getConnection(AbstractMetaDataDialect.java:122)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getMetaData(AbstractMetaDataDialect.java:61)
	... 24 more
2009-05-07 02:38:12,359 [SwingWorker-pool-2-thread-8] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - Getting database metadata
org.hibernate.exception.SQLGrammarException: Getting database metadata
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:67)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getMetaData(AbstractMetaDataDialect.java:64)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.caseForSearch(AbstractMetaDataDialect.java:163)
	at org.hibernate.cfg.reveng.dialect.JDBCMetaDataDialect.getTables(JDBCMetaDataDialect.java:22)
	at org.hibernate.cfg.reveng.JDBCReader.processTables(JDBCReader.java:476)
	at org.hibernate.cfg.reveng.JDBCReader.readDatabaseSchema(JDBCReader.java:78)
	at org.hibernate.cfg.reveng.JDBCReader.readDatabaseSchema(JDBCReader.java:860)
	at org.hibernate.cfg.JDBCBinder.readDatabaseSchema(JDBCBinder.java:115)
	at org.hibernate.cfg.JDBCBinder.readFromDatabase(JDBCBinder.java:88)
	at org.hibernate.cfg.JDBCMetaDataConfiguration.readFromJDBC(JDBCMetaDataConfiguration.java:42)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.autodiscoveryMappings(HibernateSyncAdapterFactory.java:197)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createMappings(HibernateSyncAdapterFactory.java:141)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createHibernateAdapter(HibernateSyncAdapterFactory.java:58)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMySQLAdapter(SyncAdapterBuilder.java:140)
	at org.mesh4j.ektoo.controller.MySQLUIController.createAdapter(MySQLUIController.java:104)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:336)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:57)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Unknown database 'ektoo'
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.Util.getInstance(Util.java:381)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:910)
	at com.mysql.jdbc.MysqlIO.secureAuth411(MysqlIO.java:3923)
	at com.mysql.jdbc.MysqlIO.doHandshake(MysqlIO.java:1273)
	at com.mysql.jdbc.ConnectionImpl.createNewIO(ConnectionImpl.java:2031)
	at com.mysql.jdbc.ConnectionImpl.<init>(ConnectionImpl.java:718)
	at com.mysql.jdbc.JDBC4Connection.<init>(JDBC4Connection.java:46)
	at sun.reflect.GeneratedConstructorAccessor52.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.ConnectionImpl.getInstance(ConnectionImpl.java:302)
	at com.mysql.jdbc.NonRegisteringDriver.connect(NonRegisteringDriver.java:282)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.hibernate.connection.DriverManagerConnectionProvider.getConnection(DriverManagerConnectionProvider.java:110)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getConnection(AbstractMetaDataDialect.java:122)
	at org.hibernate.cfg.reveng.dialect.AbstractMetaDataDialect.getMetaData(AbstractMetaDataDialect.java:61)
	... 24 more
2009-05-07 02:38:25,609 [SwingWorker-pool-2-thread-9] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - could not load an entity: [ektoo_sync_info#6b6da7cd-6596-4fcb-8c46-7fdb434ab161]
org.hibernate.exception.SQLGrammarException: could not load an entity: [ektoo_sync_info#6b6da7cd-6596-4fcb-8c46-7fdb434ab161]
	at org.hibernate.exception.SQLStateConverter.convert(SQLStateConverter.java:67)
	at org.hibernate.exception.JDBCExceptionHelper.convert(JDBCExceptionHelper.java:43)
	at org.hibernate.loader.Loader.loadEntity(Loader.java:1874)
	at org.hibernate.loader.entity.AbstractEntityLoader.load(AbstractEntityLoader.java:48)
	at org.hibernate.loader.entity.AbstractEntityLoader.load(AbstractEntityLoader.java:42)
	at org.hibernate.persister.entity.AbstractEntityPersister.load(AbstractEntityPersister.java:3049)
	at org.hibernate.event.def.DefaultLoadEventListener.loadFromDatasource(DefaultLoadEventListener.java:399)
	at org.hibernate.event.def.DefaultLoadEventListener.doLoad(DefaultLoadEventListener.java:375)
	at org.hibernate.event.def.DefaultLoadEventListener.load(DefaultLoadEventListener.java:139)
	at org.hibernate.event.def.DefaultLoadEventListener.proxyOrLoad(DefaultLoadEventListener.java:195)
	at org.hibernate.event.def.DefaultLoadEventListener.onLoad(DefaultLoadEventListener.java:103)
	at org.hibernate.impl.SessionImpl.fireLoad(SessionImpl.java:878)
	at org.hibernate.impl.SessionImpl.get(SessionImpl.java:815)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncRepository.getSyncInfo(HibernateSyncRepository.java:63)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncRepository.get(HibernateSyncRepository.java:41)
	at org.mesh4j.sync.adapters.split.SplitAdapter.get(SplitAdapter.java:130)
	at org.mesh4j.sync.SyncEngine.mergeItems(SyncEngine.java:154)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:117)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:50)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:67)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:60)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: Table 'ektoo.ektoo_sync_info' doesn't exist
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
	at java.lang.reflect.Constructor.newInstance(Unknown Source)
	at com.mysql.jdbc.Util.handleNewInstance(Util.java:406)
	at com.mysql.jdbc.Util.getInstance(Util.java:381)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:1030)
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:956)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3491)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3423)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1936)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2060)
	at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2542)
	at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1734)
	at com.mysql.jdbc.PreparedStatement.executeQuery(PreparedStatement.java:1885)
	at org.hibernate.jdbc.AbstractBatcher.getResultSet(AbstractBatcher.java:186)
	at org.hibernate.loader.Loader.getResultSet(Loader.java:1787)
	at org.hibernate.loader.Loader.doQuery(Loader.java:674)
	at org.hibernate.loader.Loader.doQueryAndInitializeNonLazyCollections(Loader.java:236)
	at org.hibernate.loader.Loader.loadEntity(Loader.java:1860)
	... 27 more
2009-05-07 02:39:05,062 [SwingWorker-pool-2-thread-10] INFO  org.mesh4j.sync.translator.MessageTranslator - Resource Bundle for key <INVALID_TABLE_NAME> does not exist.
2009-05-07 02:39:05,062 [SwingWorker-pool-2-thread-10] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - INVALID_TABLE_NAME
org.mesh4j.sync.validations.MeshException: INVALID_TABLE_NAME
	at org.mesh4j.sync.validations.Guard.throwsException(Guard.java:44)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createMappings(HibernateSyncAdapterFactory.java:145)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createHibernateAdapter(HibernateSyncAdapterFactory.java:58)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMySQLAdapter(SyncAdapterBuilder.java:140)
	at org.mesh4j.ektoo.controller.MySQLUIController.createAdapter(MySQLUIController.java:104)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:336)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:57)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2009-05-07 02:39:12,828 [SwingWorker-pool-2-thread-11] INFO  org.mesh4j.sync.translator.MessageTranslator - Resource Bundle for key <INVALID_TABLE_NAME> does not exist.
2009-05-07 02:39:12,828 [SwingWorker-pool-2-thread-11] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - INVALID_TABLE_NAME
org.mesh4j.sync.validations.MeshException: INVALID_TABLE_NAME
	at org.mesh4j.sync.validations.Guard.throwsException(Guard.java:44)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createMappings(HibernateSyncAdapterFactory.java:145)
	at org.mesh4j.sync.adapters.hibernate.HibernateSyncAdapterFactory.createHibernateAdapter(HibernateSyncAdapterFactory.java:58)
	at org.mesh4j.ektoo.SyncAdapterBuilder.createMySQLAdapter(SyncAdapterBuilder.java:140)
	at org.mesh4j.ektoo.controller.MySQLUIController.createAdapter(MySQLUIController.java:104)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:336)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:57)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2009-05-07 21:22:03,390 [SwingWorker-pool-2-thread-1] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - left
java.lang.IllegalArgumentException: left
	at org.mesh4j.sync.validations.Guard.argumentNotNull(Guard.java:15)
	at org.mesh4j.sync.SyncEngine.<init>(SyncEngine.java:33)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:62)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:57)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2009-05-07 21:46:57,625 [SwingWorker-pool-2-thread-1] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - googleSpreadSheetId
java.lang.IllegalArgumentException: googleSpreadSheetId
	at org.mesh4j.sync.validations.Guard.argumentNotNull(Guard.java:15)
	at org.mesh4j.sync.validations.Guard.argumentNotNullOrEmptyString(Guard.java:25)
	at org.mesh4j.ektoo.GoogleSpreadSheetInfo.<init>(GoogleSpreadSheetInfo.java:25)
	at org.mesh4j.ektoo.controller.GSSheetUIController.createAdapter(GSSheetUIController.java:76)
	at org.mesh4j.ektoo.ui.SyncItemUI.createAdapter(SyncItemUI.java:352)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:54)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
2009-05-07 21:50:20,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.translator.MessageTranslator - Resource Bundle for key <rdfXml> does not exist.
2009-05-07 21:50:20,546 [SwingWorker-pool-2-thread-1] ERROR org.mesh4j.ektoo.tasks.SynchronizeTask - java.lang.IllegalArgumentException: rdfXml
org.mesh4j.sync.validations.MeshException: java.lang.IllegalArgumentException: rdfXml
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.convertXMLToRow(HibernateContentAdapter.java:126)
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.save(HibernateContentAdapter.java:65)
	at org.mesh4j.sync.adapters.split.SplitAdapter.add(SplitAdapter.java:54)
	at org.mesh4j.sync.SyncEngine.importItems(SyncEngine.java:188)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:122)
	at org.mesh4j.sync.SyncEngine.synchronize(SyncEngine.java:50)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:64)
	at org.mesh4j.ektoo.controller.EktooUIController.sync(EktooUIController.java:57)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:43)
	at org.mesh4j.ektoo.tasks.SynchronizeTask.doInBackground(SynchronizeTask.java:1)
	at javax.swing.SwingWorker$1.call(Unknown Source)
	at java.util.concurrent.FutureTask$Sync.innerRun(Unknown Source)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at javax.swing.SwingWorker.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.lang.IllegalArgumentException: rdfXml
	at org.mesh4j.sync.validations.Guard.throwsArgumentException(Guard.java:36)
	at org.mesh4j.sync.payload.schema.rdf.RDFInstance.buildFromRDFXml(RDFInstance.java:79)
	at org.mesh4j.sync.payload.schema.rdf.RDFSchema.createNewInstanceFromRDFXML(RDFSchema.java:253)
	at org.mesh4j.sync.adapters.hibernate.mapping.HibernateToRDFMapping.convertXMLToRow(HibernateToRDFMapping.java:59)
	at org.mesh4j.sync.adapters.hibernate.HibernateContentAdapter.convertXMLToRow(HibernateContentAdapter.java:124)
	... 16 more
2009-05-07 21:54:54,234 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 21:54:57,437 [SwingWorker-pool-2-thread-2] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 21:55:02,515 [SwingWorker-pool-2-thread-3] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 21:55:44,062 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 21:55:51,734 [SwingWorker-pool-2-thread-2] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 21:55:53,593 [SwingWorker-pool-2-thread-3] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 21:56:06,000 [SwingWorker-pool-2-thread-4] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 21:56:55,453 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: columns_priv
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: db
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: func
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: help_category
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: help_keyword
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: help_relation
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: help_topic
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: host
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: proc
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: procs_priv
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: tables_priv
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone_leap_second
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone_name
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone_transition
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone_transition_type
2009-05-07 21:56:55,484 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: user
2009-05-07 21:56:55,500 [SwingWorker-pool-2-thread-5] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: user_info
2009-05-07 21:57:24,843 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Listing all table name in Database!
2009-05-07 21:57:25,500 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: columns_priv
2009-05-07 21:57:25,500 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: db
2009-05-07 21:57:25,500 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: func
2009-05-07 21:57:25,500 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: help_category
2009-05-07 21:57:25,500 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: help_keyword
2009-05-07 21:57:25,500 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: help_relation
2009-05-07 21:57:25,500 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: help_topic
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: host
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: proc
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: procs_priv
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: tables_priv
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone_leap_second
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone_name
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone_transition
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: time_zone_transition_type
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: user
2009-05-07 21:57:25,515 [SwingWorker-pool-2-thread-1] INFO  org.mesh4j.sync.utils.SqlDBUtils - Table: user_info
